flow/Knobs.cpp

```diff
@@ -213,6 +213,7 @@ void FlowKnobs::initialize(bool randomize, bool isSimulated) {
 	init( BASIC_LOAD_BALANCE_UPDATE_RATE,                     10.0 ); //should be longer than the rate we log network metrics
 	init( BASIC_LOAD_BALANCE_MAX_CHANGE,                      0.10 );
 	init( BASIC_LOAD_BALANCE_MAX_PROB,                         2.0 );
+	init( BASIC_LOAD_BALANCE_MIN_AMOUNT,                     50000 ); //Do not update probabilities if the average proxy busyness is less than 5%
```

Comment:
Changing the wording here to make it clear this is not an instruction to a developer editing this line.

```suggestion
	init( BASIC_LOAD_BALANCE_MIN_AMOUNT,                     50000 ); //Will not update probabilities if the average proxy busyness is less than 5%
```